.github/stale.yml

```diff
@@ -0,0 +1,25 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
```

Comment:
I'm not sure we should auto close stale issues.

Reply:
if we don't close, issues might get piled up. Contributors can always reopen them or keep them active by making a comment.